fire/cli/niv/_læs_observationer.py

```diff
@@ -330,7 +331,7 @@ def læs_observationsstrenge(
                     "Type": fil.Type.upper(),
                     "uuid": "",
                 }
-                observationer = observationer.append(obs, ignore_index=True)
+                observationer = frame.append(observationer, obs)
```

Comment:
Jeg kan ikke huske hvad `ignore_index=True` gør i den oprindelige kode, men jeg går ud fra at du har testet af `frame.append(...)`  gør det rigtige, også i dette tilfælde

Reply:
Ja, den er altid sat i kodebasen. Den kan sættes, men den bliver altid sat til True, hvis operationen kræver det. Mere fleksibelt er det altså ikke. Men vi skal også bare slipper for at skulle genopfinde den samme linje kode, hver gang. Så til eget brug vil jeg mene, at det er fint at gemme den væk. (På den anden side, så har modulet, som nævnt, en vis antydning af at være noget, amn kan generalisere ud; men i så fald skal det nok med i overvejelserne igen, om det er godt at tvinge ignore_index=True.)